test/mailers/mailer_test.rb

```diff
@@ -1,24 +0,0 @@
-require "test_helper"
```

Comment:
Why are we removing this?

Reply:
It was testing only the recipient logic:

1. it is sent only to "notifiable users"
2. it is not sent at all when no "notifiable user" is found

This logic was moved out of mailer to pusher.